frontend/src/metabase/entities/search.js

```diff
@@ -67,3 +67,9 @@ export default createEntity({
     return false;
   },
 });
+
+// change the default getObject to getObjectDenormalized
+// WARNING: this could have performance issues
+Search.selectors.getObject = Search.selectors.getObjectDenormalized;
```

Comment:
Would it be too redundant to pass `denormalized` in the loaders?

Reply:
The reason I did this is the `search` entity doesn't work at all in non-denormalized mode.